toolchain/sem_ir/function.h

```diff
@@ -186,41 +186,50 @@ using FunctionStore = ValueStore<FunctionId, Function>;
 class File;
 
 struct CalleeFunction : public Printable<CalleeFunction> {
-  // The function. `None` if not a function.
-  FunctionId function_id;
-  // The overload set, if this is a C++ overload set rather than a function.
-  CppOverloadSetId cpp_overload_set_id;
-  // The specific that contains the function.
-  SpecificId enclosing_specific_id;
-  // The specific for the callee itself, in a resolved call.
-  SpecificId resolved_specific_id;
-  // The bound `Self` type or facet value. `None` if not a bound interface
-  // member.
-  InstId self_type_id;
-  // The bound `self` parameter. `None` if not a method.
-  InstId self_id;
-  // True if an error instruction was found.
-  bool is_error;
+  // This is a C++ overload set.
+  struct CppOverload {
```

Comment:
```suggestion
  struct CppOverloadSet {
```
This represents an overload set, not a single overload.